Makefile.am

```diff
@@ -52,17 +67,17 @@ if ENABLE_LIBRARY
 	$(MKDIR_P) $(DESTDIR)$(libdir)/opencryptoki/stdll
 	$(MKDIR_P) $(DESTDIR)$(libdir)/pkcs11
 	cd $(DESTDIR)$(libdir)/opencryptoki && \
-		ln -fs libopencryptoki.so PKCS11_API.so
+		ln -fs libopencryptoki.$(SHLIBEXT) PKCS11_API.so
```

Comment:
Shouldn't `PKCS11_API.so` also use `$(SHLIBEXT)` ? 
`PKCS11_API.so` appears a second time a few lines below. 
Same question for `PKCS11_CCA.so` and `PKCS11_SW.so` below.